spec/openapi.yaml

```diff
@@ -480,6 +480,49 @@ paths:
         '500':
           description: An unexpected error occured.
 
+  /healthcheck/failure/notify:
```

Comment:
renamed to follow idiomatic rest approach of "biggest topic" -> "smaller topic" -> "action" for paths